camera_name: cam_1
image_width: 1000
image_height: 480
camera_matrix:
  rows: 3
  cols: 3
  data: [554.25, 0.0, 320.5, 0.0, 554.25, 240.5, 0.0, 0.0, 1.0]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [k1, k2, k3, t1, t2]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0]
projection_matrix:
  rows: 3
  cols: 4
  data: [554.25, 0.0, 320.5, 0.0, 0.0, 554.25, 240.5, 0.0, 0.0, 0.0, 1.0, 0.0]
roi:
  do_rectify: true
  x_offset: 0
  y_offset: 0
  height: 0
  width: 0
